src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  font-size: 10px;
+  min-width: 1000px;
+  width: getWidth(4, $cards-width, $cards-vertical_shift);
+  margin: 0 auto;
+
+  &__container {
```

Comment:
1) upper cards row shouldn't have margin-top
2) first card in each row shouldn't have margin-left
3) last card in each row shouldn't have margin-right
<img width="917" alt="Screenshot 2023-04-06 at 20 51 57" src="https://user-images.githubusercontent.com/117587463/230457252-33b29811-c949-45ae-a697-f1cbe6d26478.png">
<img width="883" alt="Screenshot 2023-04-06 at 20 50 04" src="https://user-images.githubusercontent.com/117587463/230457062-6e821b67-9a84-41f8-a263-bc2c23d1510a.png">

Reply:
If I change this, my whole project will be destroyed!